lib/builder/webpack/base.js

```diff
@@ -313,6 +327,6 @@ export default class WebpackBaseConfig {
     }
 
     // Clone deep avoid leaking config between Client and Server
-    return _.cloneDeep(config)
+    return _.cloneDeep(this.customize(config))
```

Comment:
Not related to this PR. but can we rename `customize` to `extendConfig`? @clarkdo